service/worker/indexer/processor.go

```diff
@@ -234,10 +236,21 @@ func (p *indexProcessor) generateESDoc(msg *indexer.Message, keyToKafkaMsg strin
 	return doc
 }
 
+func (p *indexProcessor) decodeSearchAttrBinary(bytes []byte, key string) interface{} {
```

Comment:
Can we rely on Thrift encode/decode rather than decoding payload ourself.

Reply:
keep as discussed